compilers/concrete-compiler/compiler/lib/Conversion/FHETensorOpsToLinalg/TensorOpsToLinalg.cpp

```diff
@@ -1934,6 +1943,9 @@ struct FHELinalgUnaryOpToLinalgGeneric
                                                  ins, outs, maps, iteratorTypes,
                                                  doc, call, bodyBuilder);
 
+    if (linalgOp->hasAttr("tile-sizes"))
+      genericOp->setAttr("tile-sizes", linalgOp->getAttr("tile-sizes"));
+
     rewriter.replaceOp(linalgOp, {genericOp.getResult(0)});
 
```

Comment:
There are some ops that is converted to linalg.generic without tile-sizes (sum, transpose, conv, partition frontier, dot) is that expected?

Reply:
Initially, this PR just added support for the operations requested in PR #748. But it makes indeed sense to extend the support to all operations from `TensorOpsToLinalg.cpp` to which tiling is applicable. The updated PR adds support for: